docs/taskversionbumping.md

```diff
@@ -0,0 +1,7 @@
+# Bumping task version
+To bump task version - please change 'version' field in task.json and task.loc.json files following the steps below:
+1. Check current Azure DevOps sprint - https://whatsprintis.it/
+2. If sprint number differs from current minor number - set it to current sprint number, set patch to 0
+3. If it's not - increase patch number
```

Comment:
For #3 it should read - "If the minor version and the sprint number number are the same"

Reply:
Thanks! Fixed it